integration-tests/test-helper.ts

```diff
@@ -1004,7 +1009,7 @@ export class TestRig {
     const options: pty.IPtyForkOptions = {
       name: 'xterm-color',
       cols: 80,
-      rows: 24,
+      rows: 80,
```

Comment:
increased the number of rows so that scrolling in the alternate buffer mode terminal doesn't result in us missing content we need for a test.